src/GitHub.App/Services/PullRequestEditorService.cs

```diff
@@ -288,6 +289,15 @@ public void OpenActiveDocumentInCodeView(IVsTextView sourceView)
             NavigateToEquivalentPosition(sourceView, codeView);
         }
 
+        public bool ConfirmCancelPendingReview()
```

Comment:
I'm still not sure this is the right place for this method. The XML doc on the class says:

```
Services for opening views of pull request files in Visual Studio.
```

Which doesn't really fit with this. I'd suggest adding it to `PullRequestSessionService`. I notice you added a `using System.Windows.Forms;` to that file, so I assume you actually started putting it there and then decided not to? Why was that?

Reply:
- `IPullRequestSessionService` cannot be imported into `PullRequestReviewAuthoringViewModel`
  - `PullRequestReviewAuthoringViewModel` lives in `GitHub.App`
  - `IPullRequestSessionService` lives in `GitHub.InlineReviews`